CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
 - Alarm raises with null error (#521)
 - Capture Mongo DB connection errors
+- Update MongoDB driver in order to fix NODE-818 error (#210)
```

Comment:
Fixed in commit https://github.com/telefonicaid/iotagent-node-lib/pull/548/commits/3f297687d3737044f515dce927a04b936f78ae30